### DISCUSSION TITLE
Welcome to LukeRobertsControl Discussions!

### DISCUSSION THREAD

**Discussion by martgras** [martgras/LukeRobertsControl#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Thaphil** [comment]
Hey martgras, 
I'm relative new to this and i have now a LukeRoberts Lamp and would like to try your Control. I already have a ESP32, but i am not able to build this, there are so many errors when i try to build the code with VS Studio & Platformio. How do you build it?
